api-docs/rst/dev-guide/glossary.rst

```diff
@@ -1,3 +1,5 @@
+.. _glossary:
+
 Glossary 
 ----------
```

Comment:
We removed the glossaries from all the dev guides and rolled the terms into the common glossary.  Can you remove this one?